utils/swift_build_support/swift_build_support/products/tensorflow.py

```diff
@@ -112,7 +132,12 @@ def build(self, host_target):
                 '-D', 'BUILD_X10={}'.format(
                     'YES' if self.args.enable_x10 else 'NO'
                 ),
-                '-D', 'X10_INCLUDE_DIR={}'.format(x10_inc),
+                '-D', 'X10_INCLUDE_DIR={}'.format(
+                    _x10_include_directory(host_target, tensorflow_source_dir)
+                ),
+                '-D', 'X10_LIBRARY={}'.format(
```

Comment:
Where is `X10_LIBRARY` used? I haven't seen any PR to swift-apis to reflect this.

Reply:
This is a staging change, to pass the variable, that will be used subsequently.  Its just passing an extra variable and adds the additional binary to the image.

Reply:
Ok, my point about `ops_test` not running still stands.

Reply:
https://github.com/tensorflow/swift-apis/pull/781 is the associated change for the swift-apis repository.

Reply:
With that change, now it doesn't even build for me, it's trying to find libx10.so under bazel-bin/tensorflow.